src/main/downloadDolphin.ts

```diff
@@ -115,8 +132,8 @@ async function installDolphin(
   assetPath: string,
   log: (message: string) => void = console.log,
 ) {
-  const dolphinPath = path.join(remote.app.getPath("userData"), type);
-  const backupLocation = path.join(remote.app.getPath("userData"), type + "_old");
+  const dolphinPath = path.join(app.getPath("userData"), type);
+  const backupLocation = path.join(app.getPath("userData"), type + "_old");
```

Comment:
Why not something like this?
```suggestion
  const dolphinPath = path.join(app.getPath("userData"), type);
  const backupLocation = dolphinPath + "_old";
```